ansible/roles/ecodata/templates/syncProdData.sh

```diff
@@ -5,10 +5,19 @@ export FILE="ecodata-$(date +%y%m%d).tgz"
 cd /data/backups/prod
 tar -xf $FILE
 # audit messages are stored outside the tar
-cp ./audit/auditMessage.* ./data/backups/dump/ecodata/
+mv ./audit/auditMessage.* ./data/backups/dump/ecodata/
+cp ./data/ecodata/models/* /data/ecodata/models/
+chown ecodata:ecodata /data/ecodata/models/*
+
 mongorestore --drop -d ecodata -u {{ ecodata_username }} -p {{ ecodata_password }} ./data/backups/dump/ecodata
 
 mongosh -u {{ ecodata_username }} -p {{ ecodata_password }} ecodata --eval 'db.setting.update({key:"meritfielddata.announcement.text"},{$set:{value:"This is the MERIT staging environment"}})';
+
+# clean up as we are tight on disk space and the audit messages are > 50GB
+mv ./data/backups/dump/ecodata/auditMessage.* ./audit/
+rm -r /data/backups/prod/data
```

Comment:
Should audit files be deleted? Looks to me audit files are not deleted. It is moved back to `/data/backups/prod/audit` in the previous step and delete is done on `/data/backups/prod/data` directory.

Reply:
I'm actually deliberately leaving it there to "reserve" the disk space for the audit file when it is re-backed up to hopefully an outage caused by filling the disk at 4am when the backup runs